.github/workflows/testing.yml

```diff
@@ -0,0 +1,43 @@
+name: Test
+on:
+  push:
+  pull_request:
+    branches:
+      - development
```

Comment:
You haven't specified any branches under push. Will it consider `development` for push and pull_request if I write it like this?

Reply:
When branches aren't specified, then all branches will be considered, including development.

Reason that I allowed push on all branches is because it allows subsequent pushes on pull requests to run the workflow. (The pull_request only runs when u make the pr, not when subsequent pushes to the pr)

Reply:
Are you sure about this? Our Lint action is listening on master and development for pushes and PRs. Albeit, I have noticed that if you push to any existing PR it will re-run Lint action it.

Reply:
My bad you are right, for some reason when I was testing it previously, it doesn't run on push to PRs. I have changed accordingly. 😅